tests/ClientService.Tests/ClientServiceTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using InvoiceGenerator.Domain;
+using InvoiceGenerator.Domain.Models;
+using InvoiceGenerator.ViewModels;
+using Moq;
+using Xunit;
+
+namespace ClientService.Tests
+{
+    public class ClientServiceTests
+    {
+        [Fact]
+        public void Check_If_Created_List_Of_Client_Details_Matches_Database_Client_Details()
+        {
+            // Arrange
+            var client = new Client()
+            {
+                ClientId = Guid.NewGuid(),
+                ClientName = "Jim",
+                ClientAddress = "10 Downing Street",
+                ContactName = "Bob",
+                ContactEmail = "bob@gmail.com"
+            };
+            var ClientsForMock = new List<Client>() { client };
+            var MockedRepository = new Mock<IRepository<Client>>();
+            MockedRepository.Setup(x => x.GetAll()).Returns(ClientsForMock);
+
+            //hint for later - for AddClients
```

Comment:
:pick: consider removing commented out code before committing. I understand this was a hint we left for some of the unit tests, but if it's not providing anything to the method that it's in consider removing it.

> Always leave the code you're editing **a little** cleaner than you found it - Robert C. Martin

The idea is that you should clean up as you go. I appreciate that we haven't discussed this yet, though.